src/widgets/rating-menu/__tests__/rating-menu-test.js

```diff
@@ -90,31 +90,6 @@ describe('ratingMenu()', () => {
     expect(helper.search).toHaveBeenCalledTimes(0);
   });
 
-  it('refines the search', () => {
```

Comment:
We do not export `_toggleRefinement` anymore from the connector.

This tested internal implementations already covered at the connector level: https://github.com/algolia/instantsearch.js/blob/bf67509fdf937f15f568ad020b860b1bff8c465d/src/connectors/rating-menu/__tests__/connectRatingMenu-test.js#L135-L141